### PR TITLE
Bd-30 feat: internationalization

### PR DESCRIPTION
**Pull request by KateTernovskaya**

### CODE REVIEW

**Review by sfgardev — CHANGES_REQUESTED** [review]

File: assets/lang-img/FlagEn.svg
[the remark this answers is not in the document]
Comment (KateTernovskaya) [reply assets/lang-img/FlagEn.svg:1]: convert svg to png

File: hocs/useTranslation.tsx
[the remark this answers is not in the document]
Comment (KateTernovskaya) [reply hocs/useTranslation.tsx:1]: done

File: public/locales/en.ts
Comment (sfgardev) [inline public/locales/en.ts:1]: you can re-export en.ts from public

File: public/locales/ru.ts
Comment (sfgardev) [inline public/locales/ru.ts:1]: you can re-export ru.ts from public

**Review by sfgardev — APPROVED** [review]

**Pull request merged**